configuration.ml

```diff
@@ -0,0 +1,112 @@
+open Utils.Json
+
+let current_version = 1
+
+type t = {
+  certificate : X509.Certificate.t;
+  private_key : X509.Private_key.t;
+  server_ip : Ipaddr.t;
+  server_port : int;
+  version : int;
+  updated_at : Ptime.t;
```

Comment:
Well, since it's a thing that can be updated it's a good idea to keep the update time. So the time here represents the timestamp when an update has been made.